filters/flowid/hashing.go

```diff
@@ -0,0 +1,37 @@
+package flowid
+
+import (
+	"crypto/rand"
```

Comment:
Do we actually need cryptographic randomness here? It seems like regular, pseudo randomness should be enough